app/bundles/CampaignBundle/Config/config.php

```diff
@@ -360,15 +360,6 @@
                 ],
             ],
         ],
-        'services' => [
-            'mautic.campaign.service.campaign'=> [
```

Comment:
We cannot simply remove a service. It can be used in some plugin by this alias. We should deprecate it like I'm suggesting in https://github.com/mautic/mautic/pull/13221/files#diff-fd1c26c007f60144a2c179150f017c31dc0e68d788f26acc7b25e1821354f805R56 but that'll have to be merged first.

Reply:
Right, this should not be removed. I've restored this config and added phpstan ignore flag for now.